feature/S3C-2729 arsenal update for 7.7

ilkescality opened the pull request:
Update the Arsenal dependency on cloudserver to make use of the new env var to customize S3 pre-sign URL duration.

bert-e commented:
# Hello ilkescality,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/cloudserver/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/8.1/feature/S3C-2729-arsenal-update-for-7.7` with contents from `feature/S3C-2729-arsenal-update-for-7.7`
and `development/8.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/8.1/feature/S3C-2729-arsenal-update-for-7.7 origin/development/8.1
 $ git merge origin/feature/S3C-2729-arsenal-update-for-7.7
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/8.1/feature/S3C-2729-arsenal-update-for-7.7
```

ilkescality committed:
feature/S3C-2729: update arsenal dependency version

Update Arsenal dependency version to make use of S3 pre-sign
URL duration configuration

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `feature/S3C-2729-arsenal-update-for-7.7` into
`development/7.7`
* [w/8.1/feature/S3C-2729-arsenal-update-for-7.7](https://github.com/scality/cloudserver/compare/development/8.1...w/8.1/feature/S3C-2729-arsenal-update-for-7.7) will be merged into `development/8.1` (pull request #2428)
* [w/8.2/feature/S3C-2729-arsenal-update-for-7.7](https://github.com/scality/cloudserver/compare/development/8.2...w/8.2/feature/S3C-2729-arsenal-update-for-7.7) will be merged into `development/8.2` (pull request #2429)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`
* `development/7.6`
* `stabilization/7.4.5`
* `stabilization/7.6.0`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @dora-korpar

* @rahulreddy

jonathan-gramain reviewed (changes requested):
Reporting my original review on integration PR which was misplaced:

For the development/7.7 merge, it looks good.

For the development/8.1 merge (the one opened by bert-e that you had to manually fix), you'll have to update the dependency hash to point to an Arsenal development/8.1 commit. For this, once you checked out the `w/8.1/...` branch as instructed by bert-e, you will need to look at the existing Arsenal commit hash on development/8.1 that contains your change (usually you can take the latest development/8.1 even if it contains other unrelated recent changes it's fine), then update the package.json with the commit hash, and re-run `yarn install` to update the dependencies in `yarn.lock` before committing the whole thing and pushing the changes.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @dora-korpar

* @rahulreddy





The following reviewers are expecting changes from the author, or must review again:

* @jonathan-gramain

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @dora-korpar

* @rahulreddy

ilkescality committed:
Merge branch 'development/7.7' into feature/S3C-2729-arsenal-update-for-7.7

ilkescality commented:
@bert-e approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @dora-korpar

* @rahulreddy










*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.7`

* :heavy_check_mark: `development/8.1`

* :heavy_check_mark: `development/8.2`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`
* `development/7.6`
* `stabilization/7.4.5`
* `stabilization/7.6.0`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.7`

* :heavy_check_mark: `development/8.1`

* :heavy_check_mark: `development/8.2`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`
* `development/7.6`
* `stabilization/7.4.5`
* `stabilization/7.6.0`



Please check the status of the associated issue S3C-2729.

Goodbye ilkescality.
